app/controllers/v0/feature_toggles_controller.rb

```diff
@@ -11,12 +11,21 @@ def index
         features_params = params[:features].split(',')
 
         features = features_params.collect do |feature_name|
-          { name: feature_name, value: Flipper.enabled?(feature_name.underscore, @current_user) }
+          underscored_feature_name = feature_name.underscore
+          { name: feature_name, value: Flipper.enabled?(underscored_feature_name, actor(underscored_feature_name)) }
         end
       else
         features = []
       end
       render json: { data: { type: 'feature_toggles', features: features } }
     end
+
+    def actor(feature_name)
+      if FLIPPER_FEATURE_CONFIG['features'].dig(feature_name, 'actor_type') == FLIPPER_ACTOR_STRING
+        Flipper::Actor.new(params[:cookie_id])
+      else
+        @current_user
+      end
```

Comment:
Can you educate me on `Flipper::Actor`? Is there any reason we shouldn't also wrap `@current_user` as well, i.e. `Flipper::Actor.new(@current_user)`? Excuse the ignorance if this is a dumb question - my knowledge of `flipper` is pretty minimal.

Reply:
user has a method called [fliper_id](https://github.com/department-of-veterans-affairs/vets-api/blob/dac70152cbd9c40ab0d70233b7e443c487e8ee07/app/models/user.rb#L307) that returns a unique id for a logged in user. 

[Flipper::Actor](https://github.com/jnunemaker/flipper/blob/master/lib/flipper/actor.rb) is a class that has the bare mininum number of methods to be n actor, just a `fliper_id` methods and an initializer.